docs/rules/require-emit-types.md

```diff
@@ -0,0 +1,61 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-emit-types
+description: require type definitions in emits
+---
+# vue/require-emit-types
```

Comment:
I noticed that the documentation doesn't describe it as a "type" validation. I think it might be better to change the rule name (e.g. `require-emit-validator`), but what do you think?

https://v3.vuejs.org/guide/component-custom-events.html#validate-emitted-events